design/notes/tokens.css

```
@@ -0,0 +1,149 @@
+:root {
+  --notes-font-serif: "Newsreader", Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)